From a004d63291b937775934e226205cd65f1726d557 Mon Sep 17 00:00:00 2001
From: rvalue <i@rvalue.moe>
Date: Fri, 12 Nov 2021 18:59:41 +0800
Subject: [PATCH] Makefile.am: fix bash-completion dependency

Target `install-data-local` might be made before bashcompletion is
ready. This change fix it.

[vivien: reword commit message]
Fixes: #402
Closes: #457
---
 Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.am b/Makefile.am
index d755a2b8..d72439c3 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -34,7 +34,7 @@ if ENABLE_BASH_COMPLETION
 bashcompletiondir = $(BASH_COMPLETION_DIR)
 bashcompletion_DATA = bash-completion
 
-install-data-local:
+install-data-local: install-bashcompletionDATA
 	( cd '$(DESTDIR)$(BASH_COMPLETION_DIR)' && mv bash-completion i3blocks )
 
 uninstall-local:
